docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -90,44 +92,44 @@ For example, the application processes only 4 video streams, so there is no need
- - Performance improvements happen at the cost of the memory footprint growth, yet the auto-batching queries the available memory (especially for the dGPUs) and limits the selected batch size accordingly.
+ - Although less critical for the throughput-oriented scenarios, the load-time with auto-batching increases by almost double.
+ - Certain networks are not safely reshapable by the "batching" dimension (specified as `N` in the layouts terms). Also, if the batching dimension is not zeroth, the auto-batching is not triggered "implicitly" by the throughput hint.
+ -  The "explicit" notion, for example, `BATCH:GPU`, uses the relaxed dimensions tracking, often making the auto-batching possible. For example, this method unlocks most **detection networks**.
+ - When *forcing* the auto-batching via the "explicit" device notion, make sure to validate the results for correctness.   
+ - Performance improvements happen at the cost of the memory footprint growth. Yet, the auto-batching queries the available memory (especially for the dGPUs) and limits the selected batch size accordingly.
 
  
 ### Configuring the Automatic Batching
-Following the OpenVINO convention for devices names, the *batching* device is named *BATCH*. The configuration options are as follows:
+Following the OpenVINO naming convention, the *batching* device is assigned the label of *BATCH*. The configuration options are as follows:
 
 | Parameter name     | Parameter description      | Default            |             Examples                                                      |
 | :---               | :---                  | :---               |:-----------------------------------------------------------------------------|
-| "AUTO_BATCH_DEVICE" | Device name to apply the automatic batching and optional batch size in brackets | N/A | "BATCH:GPU" which triggers the automatic batch size selection. Another example is the device name (to apply the batching) with directly specified batch size "BATCH:GPU(4)"     |
-| "AUTO_BATCH_TIMEOUT" | timeout value, in ms | 1000 |  you can reduce the timeout value (to avoid performance penalty when the data arrives too non-evenly) e.g. pass the "100", or in contrast make it large enough e.g. to accommodate inputs preparation (e.g. when it is serial process)     |
+| `AUTO_BATCH_DEVICE` | A device name to apply the automatic batching and optional batch size in brackets. | N/A | `BATCH:GPU` which triggers the automatic batch size selection. Another example is the device name (to apply the batching) with directly specified batch size `BATCH:GPU(4)`.     |
+| `AUTO_BATCH_TIMEOUT` | timeout value, in ms | 1000 |  To reduce the timeout value (to avoid performance penalty when the data arrives too non-evenly), for example, pass the "100" or in contrast, i.e., make it large enough to accommodate inputs preparation (e.g. when it is a serial process).     |
```

Comment:
```suggestion
| `AUTO_BATCH_TIMEOUT` | The timeout value, in ms. | 1000 |  You can reduce the timeout value to avoid performance penalty when the data arrives too unevenly). For example, set it to "100", or the contrary, i.e., make it large enough to accommodate input preparation (e.g. when it is a serial process).     |
```